mycroft/skills/nest/vocab/en-us/tempKeyword.voc

```diff
@@ -0,0 +1,4 @@
+set the thermostat to
+set the nest to
+update the nest to
+update the thermostat to
```

Comment:
Please, refer to the VolumeSkill to change this vocabulary set. We don't need **the** and **to**.